server/src/__tests__/utils/create-tag-entities.ts

```diff
@@ -0,0 +1,12 @@
+import { TagEntity } from '../../shared/infra/db/entities/tags/tag.entity'
+
+export const createMockTagEntities = (): TagEntity[] => {
+  const tags: Array<TagEntity> = []
+  for (let i = 1; i <= 3; ++i) {
+    const tagEntity = new TagEntity()
+    tagEntity.name = `Tag Name ${i}`
+
+    tags.push(tagEntity)
+  }
```

Comment:
we should probably instead have a `mockTagEntity` factory function and then return `[1,2,3].map(id => mockTagEntity(id))`, but we can defer this and do it in #70 instead